README.md

```diff
@@ -10,6 +10,10 @@ Frontend administration application for the digital marketplace.
 
 ## Quickstart
 
+System requirements include a working node.js and npm.
+
+- note that on macOS, a `brew install` of node/npm results in `Error: Cannot find module 'internal/fs'` - use nodejs.org instead
```

Comment:
This is only true for the current latest version of node and our npm packages, I don't think it's worth adding this without specifying that